micro-teloxide/src/lib.rs

```diff
@@ -0,0 +1,212 @@
+//! A minimal "mock" of teloxide crate that is needed to test macros. Changes to
+//! `teloxide`[`-core`] should be kept in sync with this... somehow
```

Comment:
Changes will definitely be out of sync if syncing is left to a human. Also, it looks pretty creepy to have this extra machinery just for testing. Storing tests and the repository being tested together is a good idea, although we need to avoid unnecessary bookkeeping. Can't we just plug in `teloxide`/`teloxide-core` with apt versions?

Reply:
We probably can 🤔  although we need to be careful not to create a circular dependency